assignment4/islandCount.py

```diff
@@ -0,0 +1,57 @@
+# Author: Li Ling
+# Last modified: 2017-06-25
+
+class IslandCounter(object):
```

Comment:
Aaa thank you! I actually didn't know that there are different python style classes!